components/leave_team_modal/leave_team_modal.jsx

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Modal} from 'react-bootstrap';
+import {FormattedMessage, injectIntl, intlShape} from 'react-intl';
+import PropTypes from 'prop-types';
+
+import WebrtcStore from 'stores/webrtc_store.jsx';
+import {WebrtcActionTypes} from 'utils/constants.jsx';
+
+class LeaveTeamModal extends React.PureComponent {
+    static propTypes = {
+
+        /**
+         * Current user id.
+         */
+        currentUserId: PropTypes.string.isRequired,
+
+        /**
+         * Current team id.
+         */
+        currentTeamId: PropTypes.string.isRequired,
+
+        /**
+         * hide action
+         */
+
+        onHide: PropTypes.func.isRequired,
+
+        /**
+         * show or hide modal
+         */
+
+        show: PropTypes.bool.isRequired,
+
+        /**
+         * is the user busy in a video call
+         */
+
+        isBusy: PropTypes.bool.isRequired,
+
+        intl: intlShape.isRequired,
+
+        actions: PropTypes.shape({
+
+            /**
+             * An action to remove user from team
+             */
+
+            removeUserFromTeam: PropTypes.func.isRequired,
+
+            /**
+             * An action to toggle the right menu
+             */
+
+            toggleSideBarRightMenu: PropTypes.func.isRequired,
+        }),
+    };
+
```

Comment:
Should add event listener on keypress to make hitting `enter` key works on this modal, could be like:
```
 import WebrtcStore from 'stores/webrtc_store.jsx';
-import {WebrtcActionTypes} from 'utils/constants.jsx';
+import Constants, {WebrtcActionTypes} from 'utils/constants.jsx';
+import {isKeyPressed} from 'utils/utils.jsx';

 class LeaveTeamModal extends React.PureComponent {
     static propTypes = {
@@ -58,6 +59,22 @@ class LeaveTeamModal extends React.PureComponent {
         }),
     };

+    componentDidMount() {
+        if (this.props.show) {
+            document.addEventListener('keypress', this.handleKeyPress);
+        }
+    }
+
+    componentWillUnmount() {
+        document.removeEventListener('keypress', this.handleKeyPress);
+    }
+
+    handleKeyPress = (e) => {
+        if (isKeyPressed(e, Constants.KeyCodes.ENTER)) {
+            this.handleSubmit(e);
+        }
+    }
+
     handleSubmit = (e) => {
         this.props.onHide();
```

Reply:
sure 👍